docs/schema/type-element-event.md

```diff
@@ -62,7 +63,7 @@ None
 
 ### Remarks
 
-The **Type** element contains an integer corresponding to one of the <span sdata="cer" target="T:Microsoft.SharePoint.SPEventReceiverType"><span class="nolink">Microsoft.SharePoint.SPEventReceiverType</span></span> values.
+The **Type** element contains an integer corresponding to one of the Microsoft.SharePoint.SPEventReceiverType values.
```

Comment:
That SPAN you removed should not leave the text as is... should be formatted as BOLD

Reply:
Comment was addressed, formatted in BOLD.